src/test/java/org/roaringbitmap/buffer/TestContainer.java

```diff
@@ -102,27 +105,27 @@ public void inotTest10() {
     assertTrue(c1 instanceof MappeableArrayContainer);
     assertEquals(14, c1.getCardinality());
     assertTrue(checkContent(c1,
-        new short[] {0, 2, 4, (short) 65190, (short) 65191, (short) 65192, (short) 65193,
-            (short) 65194, (short) 65195, (short) 65196, (short) 65197, (short) 65198,
-            (short) 65199, (short) 65200}));
+            new short[]{0, 2, 4, (short) 65190, (short) 65191, (short) 65192, (short) 65193,
```

Comment:
@lemire defined a formatting policy here https://github.com/RoaringBitmap/RoaringBitmap/tree/master/style. It's based on google check style.
HTH